### PR TITLE
Use symfony/phpunit-bridge and github actions

### PR DESCRIPTION
**Pull request by BoShurik**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,12 +19,11 @@
...
         "php" : ">=5.5.0",
-        "ext-curl": "*"
+        "ext-curl": "*",
+        "ext-json": "*"
     },
     "require-dev": {
-        "phpunit/phpunit" : "~4.0",
-        "squizlabs/php_codesniffer": "2.*",
-        "codeception/codeception": "*"
+        "symfony/phpunit-bridge" : "*"
```
Comment (rustamwin) [inline composer.json:26]: Why not use phpunit itself?

**Review by rustamwin — COMMENTED** [review]
👍

File: composer.json (hunk 1 above)
Comment (BoShurik) [reply composer.json:26]: To test against php from 5.5 to 8.2

Comment (rustamwin) [reply composer.json:26]: Are there any plans to raise the minimum PHP version according to [EOL](https://www.php.net/supported-versions.php)?

Comment (BoShurik) [reply composer.json:26]: We'll see. If it's going hard to maintain legacy php version we'll rise requirements

**Comment by bernard-ng** [comment]
Thanks for this refactoring, is it really necessary to maintain the support of PHP 5.5 to 7.4 this limits us enormously in the improvements we can make, this library would be so easy to use with the named properties and the readonly modifier etc... 

as far as possible maybe have a 3.x version with PHP 8.* support and maintain the 2.x version for PHP 5.5 to 7.4, it's a lot of work I'm aware but it could help to perpetuate the library and make upgrades possible, the telegram api is now at version 6.7 our implementation is at 6.4

**Comment by rustamwin** [comment]
Yes, there is also the option of releasing two parallel versions, but this may require additional resources.

**Pull request merged**